README.md

```diff
@@ -24,7 +24,8 @@ This software package is designed for preparing data that can be used to train g
 - [Introduction](#introduction)
 - [Input format](#input-format)
 - [End to end data preparation](#end-to-end-data-preparation)
-    - [Input Flags](#flags)
+    - [Key Flags](#key-flags)
+    - [Input Flags](#all-flags)
 - [Tokenizing one file](#tokenizing-one-file)
```

Comment:
Should we add the key parameters to this section, it also has a list of input flags. I lean towards NO because this is used rarely and it takes up a lot of real estate